Dockerfile

```diff
@@ -1,7 +1,7 @@
 
 ###################################
 #Build stage
-FROM golang:1.16-alpine3.13 AS build-env
+FROM techknowlogick/go:1.16-alpine3.13 AS build-env
```

Comment:
why base image changed to techknowlogick's one?

I want to know background.

Reply:
I see: https://github.com/go-gitea/gitea/pull/18048

Reply:
The base alpine version of go has increased to at least 1.14.

We can't use that because if we do all the docker users who are running docker <20.10.16 will find that Gitea is broken because hooks will suddenly stop running.